config/vendor_api/draft.yml

```diff
@@ -103,3 +109,84 @@ components:
           description: Optional details about why the application was rejected for the given reason.
           example: You did not attend any of the arranged interviews.
           maxLength: 65535
+    HESAData:
```

Comment:
I wonder if we should try and find a more specific label than `HESAData`, as we have other HESA data in the API (eg `hesa_degtype`) and "data" doesn’t really mean much.

`EqualityAndDiversity` maybe?